source/content/disaster-recovery.md

```diff
@@ -45,20 +46,60 @@ You can also [connect to your Redis instance](/redis/#use-the-redis-command-line
 
-While Solr can be enabled on a site with Disaster Recovery, Pantheon's Solr service is not highly available, nor does it include failover for Solr. The RTO and RPO do not apply to Pantheon's Solr Service.
+While [Solr](/solr) can be enabled on a site with Disaster Recovery, Pantheon's Solr service is not highly available, nor does it include failover for Solr. The RTO and RPO do not apply to Pantheon's Solr Service.
 
 If your site requires Solr, do not use Pantheon's Solr service. If you require a highly available Solr service, please consider an [alternative Solr service](/solr#alternatives-to-pantheons-solr-service).
 
 ### Using Pantheon Solr Service on Sites With Disaster Recovery
 
 Pantheon Solr requires additional considerations when used on sites with Disaster Recovery. In its default state, if a site with Pantheon Solr and Disaster Recovery fails over to the backup, Solr will not automatically rebuild the search index. In that case, 
 
-1. Pantheon will need to manually re-provision your Solr instance and 
-1. you will need to manually re-index Solr after failover.
+1. Pantheon will need to manually re-provision your Solr instance. In the case of failover, the support engineers will be notified via a notification ticket and they will re-provision Solr for you. 
+1. This notification will also be sent to all users with access to the live environment, and once Solr has been re-provisioned you will be notified via this support ticket.
+1. You will need to manually re-index Solr after failover. 
+
+The reindexing process is application-side, and depending on your site the process may be different.
+
+#### Drupal 8, using the search_api_pantheon module
+1. As a site administrator, navigate to https://live-some-site.pantheonsite.io/admin/config/search/search-api/server/pantheon
+1. Click “Delete all indexed data on this server” to queue all content for re-indexing.
+1. Run Drupal Cron manually until all items have been indexed. You can determine that all items are indexed when search_api stops logging to watchdog on the cron runs.
+
+#### Drupal 7 
+
+**ApacheSolr module:** You can do this at `admin/config/search/apachesolr`. Click **Queue all content for reindexing** to initiate. This will add content that has not yet been indexed to the Solr indexing queue (following the configured items-per-cron-event setting).
+
+![ApacheSolr Indexing](../images/d7-solr-reindex.png)
+
+**Search API Solr module:** Navigate to your Search Index list page at `admin/config/search/search_api` and click the index you need to rebuild. On the index view page, you can either queue all items for reindexing or clear your existing index and re-index in batches.
+
+
+#### WordPress
+
+You can initiate the reindexing process from the WordPress dashboard or via Terminus.
+1. To reindex from the WordPress Dashboard, navigate to `/wp-admin/admin.php?page=solr-power#top#solr_action` and click **Start Index**.
```

Comment:
Might need a new line before this ordered list - it's rendering as a paragraph in the preview link